### PR TITLE
COMDOX-110: Migrate PHP Tutorials from magento/devdocs

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) migrates the following PHP Tutorials from the `devdocs.magento.com` site to the Adobe DevSite:

### Tutorials

- [Custom attributes](https://devdocs.magento.com/guides/v2.4/howdoi/custom-attributes/introduction.html)
- [Customize product creation form](https://devdocs.magento.com/guides/v2.4/howdoi/customize_product.html)
- [Customize checkout](https://devdocs.magento.com/guides/v2.4/howdoi/checkout/checkout_overview.html)
- [Customize the Admin Design](https://devdocs.magento.com/guides/v2.4/howdoi/admin/customize_admin.html)

### Extension Developer Best Practices Tutorials

- [Copy fieldsets](https://devdocs.magento.com/guides/v2.4/ext-best-practices/tutorials/copy-fieldsets.html)
- [Serialized to JSON data upgrade](https://devdocs.magento.com/guides/v2.4/ext-best-practices/tutorials/serialized-to-json-data-upgrade.html)
- [Modify image library folder permissions](https://devdocs.magento.com/guides/v2.4/ext-best-practices/tutorials/modify-image-library-permissions/)
- [Creating an Admin page](https://devdocs.magento.com/guides/v2.4/ext-best-practices/extension-coding/example-module-adminpage.html)
- [Creating a dynamic row config](https://devdocs.magento.com/guides/v2.4/ext-best-practices/tutorials/dynamic-row-system-config.html)
- [Creating a custom import entity](https://devdocs.magento.com/guides/v2.4/ext-best-practices/tutorials/custom-import-entity.html)
- [Creating a custom widget](https://devdocs.magento.com/guides/v2.4/ext-best-practices/tutorials/custom-widget.html)
- [Creating an access control list (ACL) rule](https://devdocs.magento.com/guides/v2.4/ext-best-practices/tutorials/create-access-control-list-rule.html)
- [Creating an integration with an external API](https://devdocs.magento.com/guides/v2.4/ext-best-practices/tutorials/create-integration-with-api.html)
- [Create a custom REST API](https://devdocs.magento.com/guides/v2.4/ext-best-practices/tutorials/create-custom-rest-api.html)

**Staging**: https://developer-stage.adobe.com/commerce/php/tutorials/

## Affected pages

- Several

## Links to related PRs or Jira tickets

- COMDOX-110
- https://github.com/magento-commerce/devdocs/pull/3003

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My changes follow the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by keharper — COMMENTED** [review]
Other than the adjustments to the H3s in `create-admin-page.md` I think I found a couple of nits.

I'd like to turn your attention to the left nav--it's chaos. It needs some sort of organization. The tutorials originally in the `howdoi` directory were identified as "customization" tutorials, but I suggest creating the following groupings:

*Admin development*
Create an Admin page
Create a dynamic row system config
Add a text field attribute
Customize the Admin design
Customize the product creation form

*Frontend development*
Customize checkout
Create a custom widget

*Backend development*
Create an ACL rule
Create a custom import entity
Create a custom REST API
Create an integration with an external API
Convert serialized to JSON
Copy fieldsets
Modify media library folder permissions

Some of the tutorials touch multiple categories, so the divisions aren't perfect. I just listed these in their current order. Similar tutorials should be grouped together, and maybe sorted by complexity.

**Comment by jeff-matthews** [comment]
Chaos?! Hey, I don't get credit for alphabetizing them? Just kidding 😄 . Good point. I'll see what I can do.

File: src/data/navigation/sections/tutorials.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    {
+      title: "Introduction",
+      path: "/tutorials/",
+    },
+    {
+      title: "Admin development",
+      path: "/tutorials/admin/",
+      pages: [
+        {
+          title: "Create an Admin page",
```
Comment (keharper) [inline src/data/navigation/sections/tutorials.js:11]: Recommend that the order be

1. Create an Admin page
2. Customize the Admin design
3. Customize the product creation form
4. Create a dynamic row
5. Create a text field attribute

**Review by keharper — COMMENTED** [review]
Thanks for taking the time and effort to redo the directory structure, enabling the TOC to have subsections. This looks a lot better.

**Review by keharper — APPROVED** [review]

**Pull request merged**